composer.json

```
@@ -59,11 +60,13 @@
         ],
         "check:composer:config": "Build/Scripts/runTests.sh -n -s composerNormalize",
         "check:php": [
+            "@check:php:lint",
             "@check:php:lint",
             "@check:php:cs"
         ],
         "check:php:cs": "Build/Scripts/runTests.sh -n -s cgl",
         "check:php:lint": "Build/Scripts/runTests.sh -s lint",
+        "check:rector": ".Build/bin/rector process --dry-run",
```

```suggestion
        "check:php:rector": ".Build/bin/rector process --dry-run --clear-cache",
```

Sometimes a rector run is green locally but not on CI because of local caching. To avoid this, I propose to clear caches. On this small project this should have less impact.